source/_integrations/reolink.markdown

```diff
@@ -166,37 +166,43 @@ The latest firmware can be downloaded from the [Reolink download center](https:/
 
 The following models have been tested and confirmed to work:
 
-- C1 Pro
-- C2 Pro
-- CX410
-- E1 Zoom
-- E1 Outdoor
-- E1 Outdoor Pro
-- RLC-410
-- RLC-410W
-- RLC-411
-- RLC-420
-- RLC-510A
-- RLC-511
-- RLC-511W
-- RLC-520
-- RLC-520A
-- RLC-522
-- RLC-810A
-- RLC-811A
-- RLC-81PA
-- RLC-820A
-- RLC-822A
-- RLC-823A
-- RLC-833A
-- RLC-1224A
-- RLN8-410 NVR
-- RLN16-410 NVR
-- RLN36 NVR
-- Reolink Duo 2 WiFi
-- Reolink Duo Floodlight PoE
-- Reolink TrackMix (PoE and Wi-Fi)
-- Reolink Video Doorbell (PoE and Wi-Fi)
+- C1 Pro*
+- C2 Pro*
+- [CX410](https://reolink.pxf.io/bamgZv)
```

Comment:
These are affiliate links, please remove those.

Reply:
That is a pitty that it is not allowed...
Have changed them to normal links